docs/index.yml

```
@@ -79,6 +79,9 @@ conceptualContent:
         - url: https://dotnet.microsoft.com/platform/community
           itemType: whats-new
           text: .NET developer community
+        - url: /communitytoolkit/
```

This link seems to be broken, I believe this is the right one

```suggestion
        - url: /dotnet/communitytoolkit/
```